app/code/Magento/InventoryLowQuantityNotification/etc/di.xml

```diff
@@ -14,4 +14,18 @@
                 type="Magento\InventoryLowQuantityNotification\Model\SourceItemConfiguration\Command\Get"/>
     <preference for="Magento\InventoryLowQuantityNotificationApi\Api\DeleteSourceItemConfigurationInterface"
                 type="Magento\InventoryLowQuantityNotification\Model\SourceItemConfiguration\Command\Delete"/>
+    <type name="Magento\Catalog\Block\Adminhtml\Rss\NotifyStock\DescriptionProvider">
+        <plugin name="rss_adapt_description_provider"
+                type="Magento\InventoryLowQuantityNotification\Plugin\Rss\AdaptDescriptionProvider"/>
+    </type>
+    <!-- LowStockCondition -->
```

Comment:
Looks like we don't need this configuration

Reply:
Yes. Fixed.